src/Sylius/Component/Core/Model/AdjustmentInterface.php

```diff
@@ -16,8 +16,10 @@
 interface AdjustmentInterface extends BaseAdjustmentInterface
 {
     // Labels for tax, shipping and promotion adjustments.
-    const TAX_ADJUSTMENT = 'tax';
-    const SHIPPING_ADJUSTMENT = 'shipping';
-    const ORDER_PROMOTION_ADJUSTMENT = 'order_promotion';
     const ORDER_ITEM_PROMOTION_ADJUSTMENT = 'order_item_promotion';
+    const ORDER_PROMOTION_ADJUSTMENT = 'order_promotion';
+    const ORDER_SHIPPING_PROMOTION_ADJUSTMENT = 'order_shipping_promotion';
```

Comment:
Ok, I've only included the small change in the `OrderShipmentTaxesApplicator`. This will naturally break the accompanying PHPSpec for that class as there is no implementation of `Order::getShippingTotal` on this branch.